utils/sv_IO_panel_tools.py

```diff
@@ -283,7 +283,11 @@ def create_dict_of_tree(ng, skip_set={}, selected=False, identified_node=None):
 
         IsMonadInstanceNode = (node.bl_idname.startswith('SvGroupNodeMonad'))
 
-        for k, v in node.items():
+        if save_defaults:
+            node_props = ((k, getattr(node, k)) for k in node.bl_rna.__annotations__.keys())
```

Comment:
can i ask you to turn this into a function, which skips over pointers.
```python
node_props = get_node_annotations(node, pointers=False)
```
if this is to be used by nodes in general

Reply:
Ok